docs/conf.py

```diff
@@ -91,6 +91,7 @@
     'pages/*/notebooks/*-solutions.ipynb',
     'pages/*/notebooks/*-template.ipynb',
     'pages/2019_*/notebooks/bandstructure.ipynb',
+    'pages/2020_*/notebooks/bandstructure.ipynb',
```

Comment:
You don't need to ignore this, just add it in an index, and have sphinx render it also to view in the documentation

Reply:
add it in a toctree at the bottom of ` docs/pages/2020_Intro_Week/sections/bands.rst`, like we do in https://github.com/aiidateam/aiida-tutorials/blob/master/docs/pages/2020_Intro_Week/sections/data.rst

Reply:
Added it